circle.yml

```diff
@@ -1,7 +1,7 @@
 ---
 machine:
   ruby:
-    version: '2.3.3'
+    version: '2.4.1'
```

Comment:
I'm not sure, we would need to migrate to Circle CI 2.0. I will check if it is possible.